server/market/market_test.go

```diff
@@ -726,30 +727,30 @@ func TestMarket_Run(t *testing.T) {
 	// Submit order before market starts running
 	err = mkt.SubmitOrder(oRecord)
 	if err == nil {
-		t.Error("order submitted to stopped market")
+		t.Error("order successfully submitted to stopped market")
 	}
 	if !errors.Is(err, ErrMarketNotRunning) {
 		t.Fatalf(`expected ErrMarketNotRunning ("%v"), got "%v"`, ErrMarketNotRunning, err)
 	}
 
 	mktStatus := mkt.Status()
 	if mktStatus.Running {
-		t.Errorf("Market should not be running yet")
+		t.Fatalf("Market should not be running yet")
 	}
 
 	mkt.waitForEpochOpen()
 
 	mktStatus = mkt.Status()
 	if !mktStatus.Running {
-		t.Errorf("Market should be running now")
+		t.Fatalf("Market should be running now")
 	}
 
 	// Submit again
 	oRecord = newOR()
 	storMsgPI(oRecord.msgID, pi)
 	err = mkt.SubmitOrder(oRecord)
 	if err != nil {
-		t.Error(err)
+		t.Fatal(err)
```

Comment:
All of these Errors needed to be Fatals because the rest of the test won't work (and the channel receives a couple lines down may hang).